src/main/java/teammates/common/datatransfer/attributes/AdminEmailAttributes.java

```diff
@@ -15,37 +16,49 @@
 import teammates.storage.entity.AdminEmail;
 
 public class AdminEmailAttributes extends EntityAttributes<AdminEmail> {
-
-    public String emailId;
+    // Required fields
     public List<String> addressReceiver;
     public List<String> groupReceiver;
     public String subject;
+    public Text content;
+
+    // Optional fields
     public Date sendDate;
     public Date createDate;
-    public Text content;
+    public String emailId;
     public boolean isInTrashBin;
 
-    public AdminEmailAttributes(AdminEmail ae) {
-        this.emailId = ae.getEmailId();
-        this.addressReceiver = ae.getAddressReceiver();
-        this.groupReceiver = ae.getGroupReceiver();
-        this.subject = ae.getSubject();
-        this.sendDate = ae.getSendDate();
-        this.createDate = ae.getCreateDate();
-        this.content = ae.getContent();
-        this.isInTrashBin = ae.getIsInTrashBin();
-    }
-
-    public AdminEmailAttributes(String subject,
-                                List<String> addressReceiver,
-                                List<String> groupReceiver,
-                                Text content,
-                                Date sendDate) {
-        this.subject = subject;
-        this.addressReceiver = addressReceiver;
-        this.groupReceiver = groupReceiver;
-        this.content = content;
-        this.sendDate = sendDate;
+    AdminEmailAttributes() {
+        createDate = Const.TIME_REPRESENTS_DEFAULT_TIMESTAMP;
+        emailId = Const.ParamsNames.ADMIN_EMAIL_ID;
+        isInTrashBin = false;
+    }
+
+    /**
+     * Creates a new AdminEmail with default values for optional fields.
+     *
+     * <p>Following default values are set to corresponding attributes:
+     * <ul>
+     * <li>{@code false} for {@code isInTrashBin}</li>
```

Comment:
But keep this.

Reply:
As we are keeping Javadoc for ```isInTrashBin```, should we also keep it for ```sendDate```?  
```<li>{@code null} for {@code sendDate}</li>```

Reply:
@BrianCoveney sounds like a good idea.